funcs/func_misformatted.txt

```diff
@@ -0,0 +1,7 @@
+def sum(x: float, y: float) -> float:
```

Comment:
We can remove this file. If you want examples of what is good formatting and what is bad formatting add it to a contributing.md file

Reply:
Removed